python/src/aiconfig/editor/client/src/Editor.tsx

```diff
@@ -141,8 +141,115 @@ export default function Editor() {
   );
 
   return (
-    <div>
-      <MantineProvider withGlobalStyles withNormalizeCSS>
+    <div className="editorBackground">
+      <MantineProvider
+        withGlobalStyles
+        withNormalizeCSS
+        theme={{
+          colorScheme: "dark",
```

Comment:
Shouldn't this be based on the user's system theme?

Reply:
For starters We want to force the AiConfig Brand until i add the light mode colors.